modules/thermal_hydraulics/doc/content/source/components/HSCoupler2D3D.md

```diff
@@ -0,0 +1,194 @@
+# HSCoupler2D3D
+
+This component is used to couple a [HeatStructureCylindrical.md] and a
+[HeatStructureFromFile3D.md] via gap conduction and radiation.
+
+## Formulation
+
+### Overview
+
+Heat fluxes are computed at each quadrature point of each face on the 3D heat
+structure boundary. These heat fluxes then make contributions to the 2D heat structure side:
+
+!equation
+q_z = -\frac{1}{A_z} \sum\limits_\theta q_{z,\theta} A_{z,\theta} \,,
+
+where
+
+- $q_z$ is the heat flux on the 2D heat structure boundary at the
```

Comment:
$q_z$ is a linear heat rate. Maybe use $q'$ for linear heat rates, and $q''$ for the heat flux on the 3D side

Reply:
I want to keep the documentation using heat fluxes. I consider the use of linear heat rate in the code to be a (temporary) implementation detail, not part of the theory.